### ISSUE TITLE
Videos don't play in Edge on Windows

### ISSUE THREAD

**Issue by alangrainger** [alangrainger/immich-public-proxy#30]
Reported here: https://github.com/alangrainger/immich-public-proxy/issues/25#issuecomment-2485369931

**Comment by alangrainger** [comment]
@helangen I have tested this on Win 10 with Edge, and videos play for me without issues. (Any jerkiness you see here is just from the recording):

https://imgur.com/U1bnf6i

I realise you've already tested this in private browsing window, but can you check the source to look for the `playsinline` attribute? Perhaps it is cached somewhere upstream of you.

**Comment by helangen** [comment]
Yeah, checked as you suggested and it says playsinline. ![image](https://github.com/user-attachments/assets/12fcf351-91b6-4d44-84ef-984b836e3780)

**Comment by scalerow** [comment]
Im experiencing the same issue in Firefox and Chrome on Windows 11

**Comment by alangrainger** [comment]
Ahh, I'm a victim of my own 1.5.0 release. It moved the `/video` path to `/share/video`, and I didn't update my Cloudflare rule to **not** cache, as I mention [in the troubleshooting docs](https://github.com/alangrainger/immich-public-proxy?tab=readme-ov-file#troubleshooting).

So if you're also using Cloudflare, just exclude the video path from the cache and you'll solve the issue.

Please let me know if that solves it at your end.

**Comment by helangen** [comment]
I have a grey cloud for IPP and don't pass it through CF.

**Comment by alangrainger** [comment]
And can you confirm that the videos on these two links do not play?

I am not able to get these to fail on Win 10 or Win 11 with either Edge or Chrome. They work perfectly for me after fixing the mentioned Cloudflare issue (which was affecting these specific links).

- [Direct video link](https://immich-demo.note.sx/share/video/GjFe7Fr6RIaD2oE0t58DKrc9PJd9ukSQNQskTNGFTyfIXMFprv179wsfyoSJpc8SQDU/5ffd4a49-e556-4220-99ee-e6b4c645b25d)
- [Gallery link](https://immich-demo.note.sx/share/GjFe7Fr6RIaD2oE0t58DKrc9PJd9ukSQNQskTNGFTyfIXMFprv179wsfyoSJpc8SQDU)

**Comment by helangen** [comment]
I will check these links now and let you know.

**Comment by helangen** [comment]
I've tested the links on Win10 with Edge and can confirm that both work. The video plays fine.

**Comment by helangen** [comment]
I'm guessing the cache issue is not applicable when your are using CF with a grey cloud?

**Comment by helangen** [comment]
I tried generating another share link and used some other files. They play fine on my iMac using Safari, but not on Edge on Win10. It is the same machine I used for viewing our galleries.
